examples/src/main/java/io/kubernetes/client/examples/YamlExample.java

```diff
@@ -67,5 +75,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
             .endSpec()
             .build();
     System.out.println(Yaml.dump(svc));
+
+    // Read yaml configuration file, and deploy it
+    ApiClient client = Config.defaultClient();
+    Configuration.setDefaultApiClient(client);
+
+    // Example yaml file can be found in $REPO_DIR/test-sts.yaml
+    File file = new File("test-sts.yaml");
+    V1StatefulSet sts;
+
+    sts = (V1StatefulSet) Yaml.load(file);
+
+    setYamlKindMetadataForTest();
+
+    // Deployment and StatefulSet is defined in apps/v1, so you should use AppsV1Api (or later API
+    // version)
+    AppsV1Api api = new AppsV1Api();
+    V1StatefulSet createResult = api.createNamespacedStatefulSet("default", sts, null, null, null);
+
+    System.out.println(createResult);
+
+    V1Status deleteResult =
+        api.deleteNamespacedStatefulSet(
+            sts.getMetadata().getName(),
+            "default",
+            null,
+            new V1DeleteOptions(),
+            null,
+            null,
+            null,
+            null);
+    System.out.println(deleteResult);
+
+    System.exit(0);
```

Comment:
remove this line

Reply:
Throws `IllegalThreadStateException` if not exit, so #603 suggests `System.exit(0);` for incomplete. anyway, I'll try to remove it.

Reply:
mmmm it doesnt make a difference AFAIK.. i guess?